src/EditorFeatures/CSharpTest/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionTests.cs

```diff
@@ -696,6 +739,230 @@ void M(int i)
             _ => throw null,
         };
     }
+}");
+        }
+
+        [WorkItem(37947, "https://github.com/dotnet/roslyn/issues/37947")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertSwitchStatementToExpression)]
+        public async Task TestMultiLabelWithDefault()
+        {
+            await TestInCSharp8(
+@"using System;
+
+class Program
+{
+    public static string FromDay(DayOfWeek dayOfWeek)
+    {
+        [||]switch (dayOfWeek)
+        {
+            case DayOfWeek.Monday:
+                return ""Monday"";
+            case DayOfWeek.Friday:
+            default:
+                return ""Other"";
+        }
+    }
+}",
+@"using System;
+
+class Program
+{
+    public static string FromDay(DayOfWeek dayOfWeek)
+    {
+        return dayOfWeek switch
+        {
+            DayOfWeek.Monday => ""Monday"",
+            _ => ""Other"",
+        };
+    }
+}");
+        }
+
+        [WorkItem(37949, "https://github.com/dotnet/roslyn/issues/37949")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertSwitchStatementToExpression)]
+        public async Task TestMissingOnUseInNextStatement()
+        {
+            await TestMissingAsync(
+@"using System;
+
+class Program
+{
+    public static void Throw(int index)
+    {
+        string name = """";
+        [||]switch (index)
+        {
+            case 0: name = ""1""; break;
+            case 1: name = ""2""; break;
+        }
+        throw new Exception(name);
+    }
+}");
+        }
+
+        [WorkItem(36876, "https://github.com/dotnet/roslyn/issues/36876")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertSwitchStatementToExpression)]
+        public async Task TestDeclarationInOuterScope()
+        {
+            await TestInCSharp8(
+@"using System;
+
+class Program
+{
+    public static void Test()
+    {
+        long target;
+        try
+        {
+            [||]switch (origin)
+            {
+                case SeekOrigin.Begin:
+                    target = offset;
+                    break;
+
+                case SeekOrigin.Current:
+                    target = checked(offset + position);
```

Comment:
This test has compile errors in input, bunch of locals are not defined. I assume that was not intentional?